lib/imapoptions

```diff
@@ -2405,6 +2405,9 @@ product version in the capabilities
    option has no effect on options that are disabled at compile time
    (e.g., "regex"). */
 
+{ "sieve_folder", "#sieve", STRING, "3.3.1" }
```

Comment:
This needs to be "3.3.2" now

Reply:
Fixed

Reply:
Ironically, this needs to be 3.5.0 now... but don't worry about it, we can fix it properly when it actually lands on master, it doesn't matter in the meantime.

I'm thinking about a way to make this less of a nuisance for future PR's, and I think I have an idea to play around with.